src/hotspot/share/gc/shared/workgroup.cpp

```diff
@@ -198,20 +127,77 @@ class GangTaskDispatcher : public CHeapObj<mtGC> {
     }
   }
 };
+// Definitions of WorkGang methods.
 
-WorkGang::WorkGang(const char* name,
-                   uint  workers,
-                   bool  are_GC_task_threads,
-                   bool  are_ConcurrentGC_threads) :
-    AbstractWorkGang(name, workers, are_GC_task_threads, are_ConcurrentGC_threads),
-    _dispatcher(new GangTaskDispatcher())
-{ }
+WorkGang::WorkGang(const char* name, uint workers, bool are_GC_task_threads, bool are_ConcurrentGC_threads) :
+      _workers(NULL),
```

Comment:
Pre-existing: the indentation is completely arbitrary here. Please use something like 2 spaces.